install/openshift-apiserver/install.yaml

```diff
@@ -3,8 +3,12 @@ kind: Template
 metadata:
   name: openshift-apiserver
 parameters:
-- name: IMAGE
-  value: openshift/origin:latest
+- name: OPENSHIFT_IMAGE
+  value: openshift/origin
+- name: TAG
+  value: latest
+- name: IMAGE_PULL_POLICY
+  value: Always
```

Comment:
Does this mean I'm going to pull this image 10 times?  If we want a pull, why don't we do it before we run this command and leave everything as "ifnotpresent"?

Reply:
You're not pulling 10 times, but you will check if what you have is up to date 10 times. I'm hesitant into entering the pre-pulling business.

Reply:
yeah, I would rather always pull to make sure you run the latest images than dealing with users that run mixed set of images or some weird local cached copies